detox/src/devices/common/drivers/ios/tools/AppleSimUtils.js

```diff
@@ -72,6 +73,10 @@ class AppleSimUtils {
     return device;
   }
 
+  async _openSimulatorApp() {
+    await childProcess.execWithRetriesAndLogs(`open -a Simulator.app`, { retries: 1 });
```

Comment:
Why not `retries: 0` ?

Reply:
Oh right, it's **re**tries, I believe I was thinking of total tries :|... Fixing. :+1: